app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<%# in app/views/edit.html.erb %>
```

Comment:
Not that this code is largely repeated for the `new` view. Consider using a partial view.